bugbug/models/component_nn.py

```diff
@@ -21,22 +21,6 @@
     "Optional dependencies are missing, install them with: pip install bugbug[nn]\n"
 )
 
-try:
```

Comment:
Could we import the model modules only when load_model is called, so we avoid these changes?

Reply:
I will try to see how we could do it in a proper way